CONTRIBUTING.md

```diff
@@ -84,10 +84,10 @@ implementing features, refactoring code), otherwise you risk spending a lot of
 time working on something that the project's developers might not want to merge
 into the project.
 
-Please adhere to [JavaScript Standard Style](https://github.com/feross/standard) and any other requirements (such as test coverage).
```

Comment:
I was thinking it’s not needed to point it out, as all our tests include linting with `standard` anyway ... do you think we should leave it in anyway?